### PR TITLE
[ADD] product_uom

### PR DESCRIPTION
**Pull request by max3903**
Add missing unit of measures and their categories.

https://github.com/OCA/partner-contact/issues/335

### CODE REVIEW

**Review by damendieta — APPROVED** [review]

**Review by susport — APPROVED** [review]
👍

File: product_uom/data/product_uom_categ.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (lasley) [inline product_uom/data/product_uom_categ.xml]: odoo noupdate="1"

File: product_uom/data/product_uom_categ.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    <data noupdate="1">
+
+        <record id="product_category_energy" model="product.uom.categ">
+            <field name="name">Energy / Heat</field>
+        </record>
+
+    </data>
+</odoo>
```
Comment (lasley) [inline product_uom/data/product_uom_categ.xml]: Please add newline

**Review by lasley — APPROVED** [review]
Thanks @max3903 - few minor comments inline, nothing blocking though.

Seeing as this is a generic module, would you accept a PR adding our UOMs from medical? https://github.com/laslabs/vertical-medical/blob/release/10.0/medical_medication/data/product_uom.xml

**Comment by max3903** [comment]
@lasley Of course. Feel free to add other uom and their categories.

**Comment by lasley** [comment]
Sweet, I'll package up and submit once this is merged. Are you planning to forward port this module to 10 as well?

**Comment by max3903** [comment]
@lasley Nope. My customer is on v9.

**Comment by lasley** [comment]
K I'll perform that forward port then, after the medical stuff goes in anyways 😄

**Pull request merged**